tests/mail/tests.py

```diff
@@ -680,7 +714,7 @@ def test_encoding_alternatives(self):
         self.assertMessageHasHeaders(
             payload0,
             {
-                ("MIME-Version", "1.0"),
+                # (MIME-Version is only required in top-level headers.)
```

Comment:
Should this be removed, instead of commented out? (and below)

Reply:
I felt it was helpful to explain why removing this check is OK. (And clarify that it this isn't accidentally removing an important check.)

But I don't feel strongly either way.

Reply:
Based on some other review comments, I'll add a comment explaining why this check is unnecessary in an earlier commit, and then remove both that comment and the check in a later commit. (So the commit history will explain the change is intentional, but we won't have to keep this comment around.)